index.html

```diff
@@ -3,12 +3,44 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="../css/header.css">
-    <link rel="stylesheet" href="../css/sidebar.css">
-    <link rel="stylesheet" href="../css/footer.css">
-    <link rel="stylesheet" href="../css/general.css">
+    <link rel="stylesheet" href="./css/header.css">
+    <link rel="stylesheet" href="./css/sidebar.css">
+    <link rel="stylesheet" href="./css/footer.css">
+    <link rel="stylesheet" href="./css/general.css">
     <title>Home</title>
 </head>
 <body>
+    <div class="sidebar">
+        <div class="top">
+            <div class="logo">
+        <i class="bx bx-codepen"></i>
+
+        <ul>
+        <li>
+        <a href="#">
+        <i class="bx bxs-grid-alt"></i>
+        <span class="nav-item">Mammals</span>
+             </a>
+        <span class="tooltip">Mammals</span>
+        </li>
+        <li>
+            <a href="#">
+            <i class="bx bxs-grid-alt"></i>
+            <span class="nav-item">Reptiles</span>
+                 </a>
+            <span class="tooltip">Reptiles</span>
+            </li>
+            <li>
+                <a href="#">
+                <i class="bx bxs-grid-alt"></i>
+                <span class="nav-item">Birds</span>
+                     </a>
+                <span class="tooltip">Birds</span>
+                </li>
+                
+        </ul>
+    </div>
+
```

Comment:
Fix formatting and css needs to be implemented on the elements at least so it can be functional. 

![Screenshot 2024-11-26 154759](https://github.com/user-attachments/assets/e824a757-59a3-41db-8bae-a9916d14babf)

This is how it looks for me.

Reply:
The tutorial he was following included CSS later on so I guess we can do that in the next ticket

Reply:
Ah I see